app/controllers/sign_in_controller.rb

```diff
@@ -2,4 +2,8 @@
 
 class SignInController < ApplicationController
   skip_before_action :authenticate
+
+  def new
+    render "new"
```

Comment:
You can remove `render "new"` and it should still work.

Reply:
i think it was causing a code smell when i tried def new; end but i'll try this way

Reply:
still causes a code smell - should I change back?

Reply:
Rubocop is happy with `def new; end`

Reply:
I've removed the action which still picks up the view and passes the code smell